src/pages/detail-page/components/BackgroundIdolImage.jsx

```diff
@@ -0,0 +1,12 @@
+const BackgroundIdolImage = ({ imgSrc }) => {
+  return (
+    <div className='absolute inset-0 hidden md:block'>
+      <img src={imgSrc} alt='idol' className='h-full w-full object-cover' />
```

Comment:
alt 텍스트에 더 구체적인 설명을 추가하여 접근성을 개선할 수 있습니다.
```suggestion
      <img src={imgSrc} alt='아이돌 이미지' className='h-full w-full object-cover' />
```